x/launch/types/message_create_chain_test.go

```diff
@@ -16,6 +17,9 @@ func TestMsgCreateChain_ValidateBasic(t *testing.T) {
 	invalidGenesisChainID := sample.MsgCreateChain(sample.Address(), "", false, 0)
 	invalidGenesisChainID.GenesisChainID = "invalid"
 
+	msgInvalidMetadataLen := sample.MsgCreateChain(sample.Address(), "foo.com", false, 0)
+	msgInvalidMetadataLen.Metadata = []byte(sample.String(spntypes.DefaultMaxMetadataLength + 10))
```

Comment:
```suggestion
	msgInvalidMetadataLen.Metadata = (sample.Bytes(spntypes.DefaultMaxMetadataLength + 1)
```
Better to use `1` to have the extreme edge case